### ISSUE TITLE
pls mention what api key websites we need

### ISSUE THREAD

**Issue by tman10001** [C0MPL3XB/Discord-Token-Generator#1]

**Comment by C0MPL3XB** [comment]
https://2captcha.com/

**Comment by tman10001** [comment]
okay can i have discord link?

**Comment by rydelek11** [comment]
this is virus dont download ! 
![obraz](https://user-images.githubusercontent.com/39122430/154074737-865d1279-78df-4384-bbfc-1c8e16276047.png)
![obraz](https://user-images.githubusercontent.com/39122430/154076511-63315294-128d-425c-b549-ed3f5e5b2962.png)
![obraz](https://user-images.githubusercontent.com/39122430/154076570-9325f364-a7f3-4006-9869-6e4b13d999fd.png)

**Comment by tman10001** [comment]
thank you so much, thank God i didnt download it

**Comment by XWapixelX** [comment]
how can I add anticaptcha instead of 2captcha?

**Comment by tman10001** [comment]
> how can I add anticaptcha instead of 2captcha?

DONT ITS A VIRUS

**Comment by XWapixelX** [comment]
Yeah bro it is look that guy dmed me on discord and nuked my brother his account it’s a token logger lol

**Comment by XWapixelX** [comment]
I unplugged my wifi cable and restarted my pc so that he couldn’t get my new discord tokens or any other info and so I just wiped my pc so I’m safe now, don’t download stuff like this guys! I was desperate to download this shit before even looking

**Comment by XWapixelX** [comment]
> from this token generator or from anticaptcha?

This whole fucking thing is a token logger

**Comment by Soliidz** [comment]
> > > from this token generator or from anticaptcha?
> > 
> > 
> > ![A3AC1DB4-CB0B-476B-B455-1A5C8A0978AF](https://user-images.githubusercontent.com/99325121/154746522-10d446dd-1812-4c78-8795-2a860ebe7096.png)
> 
> shit man wtf i was staring to do it lol

Send me a

> > from this token generator or from anticaptcha?
> 
> This whole fucking thing is a token logger

send me a dm o twitter let try to find out some token generator together: Demonike#0016

**Comment by XWapixelX** [comment]
> > > > from this token generator or from anticaptcha?
> > > 
> > > 
> > > ![A3AC1DB4-CB0B-476B-B455-1A5C8A0978AF](https://user-images.githubusercontent.com/99325121/154746522-10d446dd-1812-4c78-8795-2a860ebe7096.png)
> > 
> > 
> > shit man wtf i was staring to do it lol
> 
> Send me a
> 
> > > from this token generator or from anticaptcha?
> > 
> > 
> > This whole fucking thing is a token logger
> 
> send me a dm o twitter let try to find out some token generator together: Demonike#0016

I sent a request on discord

**Comment by XWapixelX** [comment]
> reset your password instead

No that dude prompted me to restart my pc because everytime u restart ur pc or are connected to wifi he gets your new updated tokens, he didn’t know that my pc was on and wiping lol because I unplugged my wifi.

**Comment by XWapixelX** [comment]
> > > > from this token generator or from anticaptcha?
> > > 
> > > 
> > > ![A3AC1DB4-CB0B-476B-B455-1A5C8A0978AF](https://user-images.githubusercontent.com/99325121/154746522-10d446dd-1812-4c78-8795-2a860ebe7096.png)
> > 
> > 
> > shit man wtf i was staring to do it lol
> 
> Send me a
> 
> > > from this token generator or from anticaptcha?
> > 
> > 
> > This whole fucking thing is a token logger
> 
> send me a dm o twitter let try to find out some token generator together: Demonike#0016

Bro unblock me or what’s your Twitter bro. Did u delete your server?

**Comment by Soliidz** [comment]
> > > > > from this token generator or from anticaptcha?
> > > > 
> > > > 
> > > > ![A3AC1DB4-CB0B-476B-B455-1A5C8A0978AF](https://user-images.githubusercontent.com/99325121/154746522-10d446dd-1812-4c78-8795-2a860ebe7096.png)
> > > 
> > > 
> > > shit man wtf i was staring to do it lol
> > 
> > 
> > Send me a
> > > > from this token generator or from anticaptcha?
> > > 
> > > 
> > > This whole fucking thing is a token logger
> > 
> > 
> > send me a dm o twitter let try to find out some token generator together: Demonike#0016
> 
> Bro unblock me or what’s your Twitter bro. Did u delete your server?

He fucked up with my server and discord account

**Comment by Soliidz** [comment]
> > > > > > from this token generator or from anticaptcha?
> > > > > 
> > > > > 
> > > > > ![A3AC1DB4-CB0B-476B-B455-1A5C8A0978AF](https://user-images.githubusercontent.com/99325121/154746522-10d446dd-1812-4c78-8795-2a860ebe7096.png)
> > > > 
> > > > 
> > > > shit man wtf i was staring to do it lol
> > > 
> > > 
> > > Send me a
> > > > > from this token generator or from anticaptcha?
> > > > 
> > > > 
> > > > This whole fucking thing is a token logger
> > > 
> > > 
> > > send me a dm o twitter let try to find out some token generator together: Demonike#0016
> > 
> > 
> > Bro unblock me or what’s your Twitter bro. Did u delete your server?
> 
> He fucked up with my server and discord account

send me ur discord to add u

**Comment by XWapixelX** [comment]
> > > > > > > from this token generator or from anticaptcha?
> > > > > > 
> > > > > > 
> > > > > > ![A3AC1DB4-CB0B-476B-B455-1A5C8A0978AF](https://user-images.githubusercontent.com/99325121/154746522-10d446dd-1812-4c78-8795-2a860ebe7096.png)
> > > > > 
> > > > > 
> > > > > shit man wtf i was staring to do it lol
> > > > 
> > > > 
> > > > Send me a
> > > > > > from this token generator or from anticaptcha?
> > > > > 
> > > > > 
> > > > > This whole fucking thing is a token logger
> > > > 
> > > > 
> > > > send me a dm o twitter let try to find out some token generator together: Demonike#0016
> > > 
> > > 
> > > Bro unblock me or what’s your Twitter bro. Did u delete your server?
> > 
> > 
> > He fucked up with my server and discord account
> 
> send me ur discord to add u
Wapixel#0433

**Comment by cliphd** [comment]
Repository owner deleted a comment from....

**Comment by assault1337** [comment]
he got exposed LOL

**Comment by XWapixelX** [comment]
yeah this shit is a token logger lmaoo
